src/components/FilterBtn.tsx

```diff
@@ -0,0 +1,19 @@
+import { memo } from "react";
+import { filterBtn } from "../utils/models";
```

Comment:
models should start with cap letter and avoid short forms `filterBtn` -> `FilterButton`